cybersixgill_actionable_alerts/assets/dashboards/cybersixgill_actionable_alerts_overview.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
I am unable to review the dashboard. Can you re-copy the dashboard json from your account and upload it here? Thanks!

Reply:
@fshaik3494 Can you update the title on the dashboard and export it to replace the output in this file? Thanks!